packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1172,6 +1196,33 @@ class EditableText extends StatefulWidget {
   ///    and notifies its listeners on [TextEditingValue] changes.
   final ValueChanged<String>? onChanged;
 
+  /// {@template flutter.widgets.editableText.onContentCommitted}
+  /// Called when a user inserts content through the device keyboard,
```

Comment:
by "device keyboard" you mean the virtual (or onscreen) keyboard?

Reply:
Correct